istio-cni/istio-cni.spec

```diff
@@ -13,16 +13,21 @@
 %global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
 
 Name:           istio-cni
-Version:        1.1.8
+Version:        1.1.10
 Release:        1%{?dist}
 Summary:        Istio CNI Plugin
 License:        ASL 2.0
 URL:            https://%{provider_prefix}
 
 Source0:        https://%{provider_prefix}/archive/%{git_commit}/%{repo}-%{git_commit}.tar.gz
+Patch0:         0001-MAISTRA-1460-Update-yaml.v2-to-v2.2.4.patch
```

Comment:
Why are we patching it here instead of applying the changes in the repo directly?